code/modules/economy/cael/ATM.dm

```diff
@@ -326,15 +326,18 @@
 			if("e_withdrawal")
 				var/amount = max(text2num(href_list["funds_amount"]),0)
 				amount = round(amount, 0.01)
+				var/obj/item/charge_card/E
```

Comment:
Since you made bronze a subtype instead of the base type, specify the bronze path here